requirements.txt

```diff
@@ -1,3 +1,2 @@
-beautifulsoup4
-lxml
-requests
+beautifulsoup4==4.14.2
+Requests==2.32.5
```

Comment:
Is the initial capital letter correct? This was not present in the deleted dependency above.